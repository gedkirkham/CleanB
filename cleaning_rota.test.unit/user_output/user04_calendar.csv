Date,Kitchen,Bedroom,Lounge,Garage,Master Bedroom,
01/09/2018,,,,Ged,Razvan,
08/09/2018,Ged,Razvan,Ali,,Ali,
15/09/2018,Razvan,,,,Ged,
22/09/2018,Ali,Ali,,,Razvan,
29/09/2018,Ged,,,Razvan,Ali,
06/10/2018,Razvan,Ged,Ged,,Ged,
13/10/2018,Ali,,,,Razvan,
20/10/2018,Ged,Razvan,,,Ali,
27/10/2018,Razvan,,,Ali,Ged,
03/11/2018,Ali,Ali,Razvan,,Razvan,
10/11/2018,Ged,,,,Ali,
17/11/2018,Razvan,Ged,,,Ged,
24/11/2018,Ali,,,Ged,Razvan,
01/12/2018,Ged,Razvan,Ali,,Ali,
08/12/2018,Razvan,,,,Ged,
15/12/2018,Ali,Ali,,,Razvan,
22/12/2018,Ged,,,Razvan,Ali,
