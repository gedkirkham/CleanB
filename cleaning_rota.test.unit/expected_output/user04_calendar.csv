Date,Kitchen,Bedroom,Lounge,Garage,Master Bedroom
18/08/2018,,,,Ged,Razvan
25/08/2018,Ged,Razvan,Ali,,Ali
01/09/2018,Razvan,,,,Ged
08/09/2018,Ali,Ali,,,Razvan
15/09/2018,Ged,,,Razvan,Ali
22/09/2018,Razvan,Ged,Ged,,Ged
29/09/2018,Ali,,,,Razvan
06/10/2018,Ged,Razvan,,,Ali
13/10/2018,Razvan,,,Ali,Ged
20/10/2018,Ali,Ali,Razvan,,Razvan
27/10/2018,Ged,,,,Ali
03/11/2018,Razvan,Ged,,,Ged
10/11/2018,Ali,,,Ged,Razvan
17/11/2018,Ged,Razvan,Ali,,Ali
24/11/2018,Razvan,,,,Ged
01/12/2018,Ali,Ali,,,Razvan
08/12/2018,Ged,,,Razvan,Ali
